cypress/dependencies.bzl

```diff
@@ -0,0 +1,36 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+def http_archive(name, **kwargs):
+    maybe(_http_archive, name = name, **kwargs)
+
+# WARNING: any changes in this function may be BREAKING CHANGES for users
+# because we'll fetch a dependency which may be different from one that
+# they were previously fetching later in their WORKSPACE setup, and now
+# ours took precedence. Such breakages are challenging for users, so any
+# changes in this function should be marked as BREAKING in the commit message
+# and released only in semver majors.
+# This is all fixed by bzlmod, so we just tolerate it for now.
+def rules_cypress_dependencies():
+    # The minimal version of bazel_skylib we require
+    http_archive(
+        name = "bazel_skylib",
```

Comment:
may as well go to the latest
```
    http_archive(
        name = "bazel_skylib",
        sha256 = "3b620033ca48fcd6f5ef2ac85e0f6ec5639605fa2f627968490e52fc91a9932f",
        strip_prefix = "bazel-skylib-1.3.0",
        urls = ["https://github.com/bazelbuild/bazel-skylib/archive/refs/tags/1.3.0.tar.gz"],
    )
```

Reply:
I've started to generally avoid the mirror.bazel.build alternate URL as well to keep the code cleaner; it doesn't add very much value IMO

Reply:
Done